programs/ssl/ssl_pthread_server.c

```diff
@@ -514,12 +525,13 @@ int main( void )
 #if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C)
     mbedtls_memory_buffer_alloc_free();
 #endif
-
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_teardown( &platform_ctx );
+#endif
 #if defined(_WIN32)
     mbedtls_printf( "  Press Enter to exit this program.\n" );
     fflush( stdout ); getchar();
 #endif
-
     return( ret );
```

Comment:
This is in the oroiginal, but should be exit_code. or at least change this value, as the shell cannot handle errors larger than 128